hare/eligibility/oracle.go

```diff
@@ -15,7 +15,8 @@ import (
 	"sync"
 )
 
-const cacheSize = 5 // we don't expect to handle more than three layers concurrently
+const vrfMsgCacheSize = 18 // each layer have 9 layers on average hence numConcurrentLayers*avgNumRounds = 2*9 = 18
+const activesCacheSize = 5 // we don't expect to handle more than three layers concurrently
 
```

Comment:
how long until @barakshani  want's this configurable ?

Reply:
the number of rounds is constant in the protocol.
the number of concurrent layers is ideally 2. It could be more if there are consensus processes running in parallel (which is not normal).
@barakshani  how big do you want the caches to be?